calendar.go

```diff
@@ -69,19 +71,20 @@ func (c *Calendar) GetVersion() float64 {
 	return c.version
 }
 
-func (c *Calendar) SetTimezone(tz time.Location) *Calendar {
+func (c *Calendar) SetTimezone(tz *time.Location) *Calendar {
```

Comment:
Regarding all these setters and getters - they are meaningless. Those, that are just fancy wrappers around field access should be replaced with public fields instead.